vello_encoding/src/encoding.rs

```diff
@@ -431,10 +431,7 @@ impl Encoding {
 
     /// Encodes an image brush.
     pub fn encode_image<'b>(&mut self, brush: impl Into<ImageBrushRef<'b>>, alpha: f32) {
-        let brush = brush.into();
-        if brush.image.alpha_type != peniko::ImageAlphaType::Alpha {
-            unimplemented!("Unsupported image alpha type: {:?}", brush.image.alpha_type);
-        }
+        let brush: ImageBrushRef<'b> = brush.into();
```

Comment:
I wrote type explicitly for RA, or else it cannot infer type and give good suggestions.